app/presenters/shopify/part_presenter.rb

```diff
@@ -0,0 +1,30 @@
+module Shopify
+  class PartPresenter < Delegator
+    attr_accessor :permutation
+
+    def weight_unit
+      'oz'
+    end
+
+    def sku
+      return model.sku if permutation.nil?
+      generate_part_sku_from(permutation)
+    end
+
+    private
+
+    def generate_part_sku_from(permutation)
+      bundle_sku = bundle.sku
+      permutation_skus = permutation.map { |p| p[:sku] }.join(sku_separator)
+      "#{bundle_sku}#{sku_separator}#{permutation_skus}"
+    end
+
+    def bundle
+      product.master
+    end
+
+    def sku_separator
```

Comment:
I mean it's nice, sure. But do we know people who use this will even use the bundle extension? Perhaps Glossier will be the only folks who use the gem with bundled products. It feels a little like premature extension but, it's totally fine.